seq2science/rules/quantification.smk

```diff
@@ -261,7 +261,23 @@ elif config["quantifier"] == "kallistobus":
             {params.options} {input.reads} > {log} 2>&1
             """
 
-            
+    rule kb_seurat_pp:
+        input:
+            expand([f"{{result_dir}}/{{quantifier}}/{custom_assembly(treps.loc[trep, 'assembly'])}-{trep}" for trep in treps.index], **config)
+        output:
+            html=f"{config['result_dir']}/seurat/{{quantifier}}/{{assembly}}/kb_seurat_pp.html",
+            qc_dir=directory(f"{config['result_dir']}/seurat/{{quantifier}}/{{assembly}}/qc")
+        priority: 1
+        conda:
+            "../envs/kb_scrna_R_seurat3.yaml"
+        params:
+            kb_dir=f"{config['result_dir']}/kallistobus"
```

Comment:
Something that is important since seq2science version v0.5 and higher, is that seq2science now also automatically infers from which rules the params have changed, and will re-run those on a re-run. For this to work properly you need to add all the relevant params to this rule. :sleeping:

Reply:
yeah, i removed that parameter since it wasn't really used anywhere and suddenly re-running became a lot easier :)